.github/workflows/tests.yml

```diff
@@ -18,8 +18,9 @@ jobs:
     strategy:
       matrix:
         ruby-version: ['3.1.2']
+        node-version: ['16.16.0']
```

Comment:
Where do we use this?

Reply:
On the yarn install

Reply:
I'm not seeing it? I would expect `${{ matrix.node-version }}` somewhere?

Reply:
Hmmm thought this would help as the issue also was also raised from node and GitHub automatically acted on this variable.